mnefun/_scoring.py

```diff
@@ -114,7 +114,13 @@ def _read_events(p, subj, ridx, raw):
             raise RuntimeError('Non-unique event samples found in %s'
                                % (fname,))
         events.append(these_events)
-    events = concatenate_events(events, raw._first_samps, raw._last_samps)
+    if len(events) == 1 and len(raw._first_samps) > 1:  # for split raw
+        first_samps = np.array(raw._first_samps[0], ndmin=1)
```

Comment:
I've more often seen `np.atleast_1d` for these sorts of things, but here I would do something more compact:
```
first_samps = raw._first_samps[:1]
last_samps = raw_last_samps[-1:]
```

Reply:
.atleast_1d() is new to me, but the slice trick is golden.  I'll fix that and let you merge.  Thank you.